<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScaplVis</class>
 <widget class="QMainWindow" name="ScaplVis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="browse_h5ad">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>22</y>
      <width>98</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Browse h5ad file&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
   <widget class="QListView" name="gene_list">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>60</y>
      <width>261</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="file_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>File:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gene_search">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>20</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="plot_selection_umap">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>200</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> UMAP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>353</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>v 1.3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plot_selection_tsne">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>230</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> t-SNE</string>
    </property>
   </widget>
   <widget class="QLabel" name="gen_expr_label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>180</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Gene Expression</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plot_selection_violin">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>260</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> Violin</string>
    </property>
   </widget>
   <widget class="QLabel" name="file_selected">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>No File Selected</string>
    </property>
   </widget>
   <widget class="QLabel" name="gene_count">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="gene_label_count">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Gene Count:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cell_label_count">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cell Count:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cell_count">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plot_basic_umap">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> UMAP</string>
    </property>
   </widget>
   <widget class="QLabel" name="gen_plot_label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>           Plot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plot_basic_tsne">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>130</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> t-SNE</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>558</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
